src/node_crypto.cc

```diff
@@ -1974,6 +1974,37 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
 }
 
 
+// Put issuer inside the object
+static void AddIssuer(X509** cert,
+                      const STACK_OF(X509)* const peer_certs,
+                      Local<Object>* const info,
+                      Environment* const env) {
+  *cert = sk_X509_delete(peer_certs, 0);
+  while (sk_X509_num(peer_certs) > 0) {
+    int i;
+    for (i = 0; i < sk_X509_num(peer_certs); i++) {
+      X509* ca = sk_X509_value(peer_certs, i);
+      if (X509_check_issued(ca, *cert) != X509_V_OK)
+        continue;
+
+      Local<Object> ca_info = X509ToObject(env, ca);
+      (*info)->Set(env->issuercert_string(), ca_info);
```

Comment:
Feel free to switch this over to the non-deprecated overload of `Set()` if you like :)

Reply:
Ah good point, I'll update to use the function taking a context. Thanks

Reply:
Related: it would be more efficient to move these two lines out of the loop.  Likewise in `GetLastIssuedCert()`.